src/Apps/Checkout/checkoutRoutes.tsx

```diff
@@ -0,0 +1,35 @@
+import type { RouteProps } from "System/Router/Route"
+import loadable from "@loadable/component"
+
+const CheckoutApp = loadable(
+  () => import(/* webpackChunkName: "checkoutBundle" */ "./CheckoutApp"),
+  {
+    resolveComponent: component => component.CheckoutApp,
+  },
+)
+
+const CheckoutOverviewRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "exampleBundle" */ "./Routes/Welcome/CheckoutOverviewRoute"
```

Comment:
I think we can simplify and get rid of the nested `Welcome` structure for the routes.

Reply:
Also note the `exampleBundle` -- thats the name of the JS file that will be auto-generated at this route, so we'll want to name it right

Reply:
True, my bad for not noticing it, thanks for catching it!